cmd/podman/containers/exec.go

```diff
@@ -104,6 +104,68 @@ func init() {
 	validate.AddLatestFlag(containerExecCommand, &execOpts.Latest)
 }
 
+func PodmanshExec(_ *cobra.Command, args []string) error {
+	var nameOrID string
+
+	if registry.IsRemote() {
+		return fmt.Errorf("Podmansh is not supported for remote")
+	}
+
+	if len(args) == 0 && !execOpts.Latest {
```

Comment:
Since Latest will never be set, remove this check. Or return an error if set in that latest is not supported.